cmd/node/config/config.toml

```diff
@@ -213,14 +213,13 @@
 [TrieEpochRootHashStorage]
     [TrieEpochRootHashStorage.Cache]
         Name = "TrieEpochRootHashCache"
-        Capacity = 10
-        Type = "SizeLRU"
-        SizeInBytes = 5242 #5MB
+        Capacity = 1000
+        Type = "LRU"
```

Comment:
don't we want to add also limit in bytes?
I would suggest to add 500MB

Reply:
I don't think that much is needed because we only write in that persister. The read is done by an external tool. Changes cache type to `SizeLRU` of 300 MB